Gems/MotionMatching/Code/Source/KdTree.cpp

```diff
@@ -159,7 +177,8 @@ namespace EMotionFX::MotionMatching
         m_nodes.emplace_back(node);
 
         // Fill the frames array and calculate the median.
-        FillFramesForNode(node, frameDatabase, featureMatrix, features, parent, leftSide);
+        AZStd::vector<float> frameFeatureValues;
+        FillFramesForNode(node, frameDatabase, featureMatrix, localToSchemaFeatureColumns, frameFeatureValues, parent, leftSide);
```

Comment:
suggestion: We could potentially think about updating this function to return by value instead of this being an out parameter? Not a huge deal but that is often preferred now and will usually be as (if not more) efficient (due to RVO/NRVO)